README.md

```diff
@@ -170,12 +170,137 @@ Note that these transitions are just meant to redirect calls to the correspondin
+|`SubmitCustomUnBlacklistTransaction`| `proxyTokenContract : ByStr20, address : ByStr20` | Submit a new `UnBlacklist` transaction for future signoff |
+|`SubmitCustomUpdateBlacklisterTransaction`| `proxyTokenContract : ByStr20, newBlacklister : ByStr20` | Submit a new `UpdateBlacklister` transaction for future signoff |
+|`SubmitCustomLawEnforcementWipingBurnTransaction`| `proxyTokenContract : ByStr20, address : ByStr20` | Submit a new `LawEnforcementWipingBurn` transaction for future signoff |
+|`SubmitCustomBurnTransaction`| `proxyTokenContract : ByStr20, value : Uint128` | Submit a new `Burn` transaction for future signoff |
+|`SubmitCustomMintTransaction`| `proxyTokenContract : ByStr20, to : ByStr20, value : Uint128` | Submit a new `Mint` transaction for future signoff |
+|`SubmitCustomTransferTransaction`| `proxyTokenContract : ByStr20, to : ByStr20, value : Uint128` | Submit a new `Transfer` transaction for future signoff |
+|`SubmitCustomTransferFromTransaction`| `proxyTokenContract : ByStr20, from : ByStr20, to : ByStr20, value : Uint128` | Submit a new `TransferFrom` transaction for future signoff |
+|`SubmitCustomUpdateMasterMinterTransaction`| `proxyTokenContract : ByStr20, newMasterMinter : ByStr20` | Submit a new `UpdateMasterMinter` transaction for future signoff |
+|`SubmitCustomIncreaseMinterAllowanceTransaction`| `proxyTokenContract : ByStr20, minter : ByStr20, amount : Uint128` | Submit a new `IncreaseMinterAllowance` transaction for future signoff |
+|`SubmitCustomDecreaseMinterAllowanceTransaction`| `roxyTokenContract : ByStr20, minter : ByStr20, amount : Uint128` | Submit a new `DecreaseMinterAllowance` transaction for future signoff |
+|`SubmitCustomPauseTransaction`| `proxyTokenContract : ByStr20` | Submit a new `Pause` transaction for future signoff |
+|`SubmitCustomUnPauseTransaction`| `proxyTokenContract : ByStr20` | Submit a new `UnPause` transaction for future signoff |
+|`SubmitCustomProxyUpgradeToTransaction`| `proxyTokenContract : ByStr20, newImplementation : ByStr20` | Submit a new `UpgradeTo` transaction for future signoff |
+|`SubmitCustomProxyChangeAdminTransaction`| `proxyTokenContract : ByStr20, newAdmin : ByStr20` | Submit a new `ChangeAdmin` transaction for future signoff |
+
+#### Action Transitions
+
+| Name | Params | Description |
+|--|--|--|
+|`SignTransaction`| `transactionId : Uint32` | Sign off on an existing transaction. |
+|`RevokeSignature`| `transactionId : Uint32` | Revoke signature of existing transaction, if it has not yet been executed. |
+|`ExecuteTransaction`| `transactionId : Uint32` | Execute signed-off transaction. |
+
+#### Callback Transitions
+
+All the callback transitions can be categorized into two categories:
+- _acceptance callback transitions_ will be called when this multi-signature contact is the `recipient` of the transaction.
+- _initiator callback transitions_ will be called when this multi-signature contact is the `initiator` of the transaction.
```

Comment:
Add additional information that this is necessary for building Dapps on top of the stablecoins even if this is not used in the multisig contract and implemented with an empty body